dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java

```diff
@@ -48,30 +48,45 @@
 @JacksonXmlRootElement( localName = "valueType", namespace = DxfNamespaces.DXF_2_0 )
 public enum ValueType
 {
-    TEXT( String.class, true ),
-    LONG_TEXT( String.class, true ),
-    LETTER( String.class, true ),
+    TEXT( String.class,
+        ( aggregationType ) -> aggregationType.isAggregateable() && aggregationType == AggregationType.NONE ),
```

Comment:
This logic seems redundant, AggregationType.NONE already has aggregatable set to true.

Reply:
it was removed together with lambdas removing